matrix-meetings-widget/src/components/common/DateTimePickers/CalendarDayPicker.test.tsx

```diff
@@ -80,6 +81,54 @@ describe('<CalendarDayPicker>', () => {
     expect(await axe(container)).toHaveNoViolations();
   });
 
+  it('should have date calendar weekdays starting from Sunday for english locale', async () => {
+    const { container } = render(
+      <CalendarDayPicker
+        endDate="2022-02-07T23:59:59Z"
+        onRangeChange={onRangeChange}
+        startDate="2022-02-07T00:00:00Z"
+      />,
+      { wrapper: Wrapper },
+    );
+
+    await userEvent.click(
+      screen.getByRole('button', {
+        name: 'Choose date, selected date is February 7, 2022',
+      }),
+    );
+
+    expect(
+      screen.getAllByRole('columnheader').map((o) => o.textContent),
+    ).toEqual(['S', 'M', 'T', 'W', 'T', 'F', 'S']);
+
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it('should have date calendar weekdays starting from Monday for german locale', async () => {
+    setLocale('de');
+
+    const { container } = render(
+      <CalendarDayPicker
+        endDate="2022-02-07T23:59:59Z"
+        onRangeChange={onRangeChange}
+        startDate="2022-02-07T00:00:00Z"
+      />,
+      { wrapper: Wrapper },
+    );
+
+    await userEvent.click(
+      screen.getByRole('button', {
+        name: 'Choose date, selected date is February 7, 2022',
+      }),
+    );
+
+    expect(
+      screen.getAllByRole('columnheader').map((o) => o.textContent),
+    ).toEqual(['M', 'T', 'W', 'T', 'F', 'S', 'S']);
+
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
```

Comment:
would it be possible to make this more generic? such as a helper function that takes 'locale' and 'weekdays' to check for?

Reply:
I dropped this statement as it is not needed for the test and simplify a bit. In general the tests have common functions to setup/mock data, but the test statements are just straightforward here to fit how it is done to existing codebase.

Reply:
We can go with this for now - just wondering if / when there is the need to support more locales if this can scale.